### ISSUE TITLE
Property field loses focus when editing a select option value in the desktop modeler form editor 

### ISSUE THREAD

**Issue by christian-konrad** [camunda/camunda-modeler#2795]
__Describe the Bug__

After editing a select option value again in the desktop modeler form editor, the field loses focus after every keystroke. This does not happen for the label, only for the value, and only after values has already been added.

__Steps to Reproduce__

1. Create a new form
2. Add a select box
3. Add some option values
4. Fokus an option value again (that you have been defined previously)
5. Try to edit it: Entering any char at any position causes the field to lose focus

__Expected Behavior__

A keystroke should not make the field focus blur (excluding Tab, Enter etc.)

__Environment__

- OS: MacOS Monterey 12.2.1
- Camunda Modeler Version: 4.12.0
- Execution Platform: None selected for the form
- Installed plug-ins: None

**Comment by christian-konrad** [comment]
See screencast attached:
https://user-images.githubusercontent.com/56470362/156335397-9183ab32-28c2-4c8a-bf43-a125b483c12e.mov

**Comment by MaxTru** [comment]
Thanks for reporting.

This is a duplicate of https://github.com/camunda/camunda-modeler/issues/2603 and was already fixed upstream via https://github.com/bpmn-io/form-js/pull/211. 

Good catch though! :-)
